AudioDeviceExample.xcodeproj/project.pbxproj

```diff
@@ -260,6 +260,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
I think there is iOS 9 specific code in this example. Can it be removed?

Reply:
This has been removed.